test/integration/admin/programs/program/workflows.js

```diff
@@ -44,9 +43,18 @@ context('program workflows page', function() {
         fx.data = testAction;
         return fx;
       })
+      .routeProgramFlows(fx => {
```

Comment:
I think this may be flaky if the random timestamp isn't after the `updated_at`s above